### PR TITLE
support python 39, ansible-core 2.12, ansible-plugin-scan

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

File: .sanity-ansible-ignore-2.12.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+plugins/modules/blivet.py import-3.6!skip
+plugins/modules/blivet.py import-3.7!skip
+plugins/modules/blivet.py import-3.8!skip
+plugins/modules/blivet.py import-3.9!skip
+plugins/modules/blivet.py import-3.10!skip
+plugins/modules/blivet.py validate-modules:import-error
+plugins/modules/blivet.py validate-modules:missing-gplv3-license
+plugins/modules/blivet.py validate-modules:module-invalid-version-added
+plugins/modules/blockdev_info.py import-2.7!skip
+plugins/modules/blockdev_info.py import-3.5!skip
```
Comment (nhosoi) [inline .sanity-ansible-ignore-2.12.txt:13]: May I ask why blockdev_info.py, find_unused_disk.py, etc. are ok to skip just 2.7 and 3.5 (unlike blivet.py has to do all versions)?

Comment (richm) [reply .sanity-ansible-ignore-2.12.txt:13]: > May I ask why blockdev_info.py, find_unused_disk.py, etc. are ok to skip just 2.7 and 3.5 (unlike blivet.py has to do all versions)?

Good question.  I don't know why we don't get import errors with those files on other versions of python.

**Comment by richm** [comment]
ok to merge?

**Review by nhosoi — APPROVED** [review]
> ok to merge?

lgtm

(I was hoping to see the citool tests pass, but our workaround to use provision-fmf.yaml in the baseos ci was merged to the staging, but not to the production. I'd think. Anyway, that's a totally separate issue. :)

**Pull request merged**